src/components/tooltip/tooltip.spec.ts

```diff
@@ -0,0 +1,97 @@
+import {
+    it,
+    describe,
+    expect,
+    beforeEach,
```

Comment:
Once #843 goes in, the imports for jasmine functions (`it`, `beforeEach`, etc.) go away. Also `beforeEachProviders` and `provide` go away.